db/migrate/20180621093054_update_cabinet_office_stats_to_be_custom.rb

```diff
@@ -0,0 +1,19 @@
+class UpdateCabinetOfficeStatsToBeCustom < ActiveRecord::Migration[5.0]
+  def up
+    rt = ReportType.find_by_abbr('R004')
+    if rt.nil?
+      ReportType.find_or_create_by!(abbr:'R004', full_name: 'Cabinet Office report', class_name: 'Stats::R004CabinetOfficeReport', custom_report: true, seq_id: 400)
```

Comment:
Also, since this is a data migration, we should define a skeleton `ReportType` class in it with the minimum of machinery needed to make the `find_by_abbr` and `create!` work.